docs/usage/self-hosted-configuration.md

```diff
@@ -656,6 +656,14 @@ CI jobs are usually triggered by these events: pull-request creation, pull-reque
 Set as an integer.
 Default is no limit.
 
+## presetCache
+
+When this feature is enabled, resolved presets will be cached in Renovate's package cache, enabling reuse across multiple repositories.
+Doing so improves efficiency because shared presets don't need to be reloaded/resolved for every repository, however it also means that private presets can be "leaked" between repositories.
+You should only enable this when all repositories are trusted, such as a corporate environment.
+
+TTL is 15 minutes by default, and it is adjustable in [cacheTtlOverride](#cachettloverride).
+
```

Comment:
```suggestion

TTL is 15 minutes by default, and it is adjustable in [cacheTtlOverride](#cachettloverride).

<!-- prettier-ignore -->
!!! warning
     Doing so improves efficiency because shared presets don't need to be reloaded/resolved for every repository, 
     however it also means that private presets can be "leaked" between repositories.
     You should only enable this when all repositories are trusted, such as a corporate environment.
```